### ISSUE TITLE
Unable to install Firefox

### ISSUE THREAD

**Issue by satmandu** [chromebrew/chromebrew#7139]
```
chronos@localhost /usr/local/lib/crew/packages (master|SPARSE=)$ crew install firefox
firefox: Mozilla Firefox (or simply Firefox) is a free and open-source web browser
https://www.mozilla.org/en-US/firefox/
Version: 101.0
License: MPL-2.0, GPL-2 and LGPL-2.1
Performing pre-flight checks...
No precompiled binary available for your platform, downloading source...
[#################################################################################]  74.01 MB 100%
Firefox archive downloaded.
Unpacking archive using 'tar', this may take a while...
Preconfiguring package...
########################################################################################### 100.0%
Checking for FHS3 compliance...
Warning: /usr/local/firefox in firefox is not FHS3 compliant.
Checking for conflicts with files from installed packages...
Error: There is a conflict with the same file in another package.
firefox::all:/usr/local/bin/firefox
firefox::all:/usr/local/firefox/fonts/TwemojiMozilla.ttf
firefox::all:/usr/local/firefox/updater.ini
firefox::all:/usr/local/firefox/dependentlibs.list
firefox::all:/usr/local/firefox/libfreeblpriv3.so
firefox::all:/usr/local/firefox/libxul.so.sig
firefox::all:/usr/local/firefox/gmp-clearkey/0.1/libclearkey.so
firefox::all:/usr/local/firefox/gmp-clearkey/0.1/libclearkey.so.sig
firefox::all:/usr/local/firefox/gmp-clearkey/0.1/manifest.json
firefox::all:/usr/local/firefox/libipcclientcerts.so
firefox::all:/usr/local/firefox/libmozavcodec.so
firefox::all:/usr/local/firefox/crashreporter.ini
firefox::all:/usr/local/firefox/libnssutil3.so
firefox::all:/usr/local/firefox/libplc4.so
firefox::all:/usr/local/firefox/firefox.sig
firefox::all:/usr/local/firefox/Throbber-small.gif
firefox::all:/usr/local/firefox/libmozsandbox.so
firefox::all:/usr/local/firefox/firefox-bin.sig
firefox::all:/usr/local/firefox/minidump-analyzer
firefox::all:/usr/local/firefox/libssl3.so
firefox::all:/usr/local/firefox/update-settings.ini
firefox::all:/usr/local/firefox/plugin-container.sig
firefox::all:/usr/local/firefox/precomplete
firefox::all:/usr/local/firefox/crashreporter
firefox::all:/usr/local/firefox/libmozgtk.so
firefox::all:/usr/local/firefox/libmozavutil.so
firefox::all:/usr/local/firefox/libsoftokn3.so
firefox::all:/usr/local/firefox/firefox
firefox::all:/usr/local/firefox/libxul.so
firefox::all:/usr/local/firefox/browser/chrome/icons/default/default48.png
firefox::all:/usr/local/firefox/browser/chrome/icons/default/default16.png
firefox::all:/usr/local/firefox/browser/chrome/icons/default/default64.png
firefox::all:/usr/local/firefox/browser/chrome/icons/default/default128.png
firefox::all:/usr/local/firefox/browser/chrome/icons/default/default32.png
firefox::all:/usr/local/firefox/browser/features/screenshots@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/features/webcompat@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/features/formautofill@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/features/doh-rollout@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/features/webcompat-reporter@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/features/pictureinpicture@mozilla.org.xpi
firefox::all:/usr/local/firefox/browser/omni.ja
firefox::all:/usr/local/firefox/browser/crashreporter-override.ini
firefox::all:/usr/local/firefox/libnspr4.so
firefox::all:/usr/local/firefox/omni.ja
firefox::all:/usr/local/firefox/libnss3.so
firefox::all:/usr/local/firefox/defaults/pref/channel-prefs.js
firefox::all:/usr/local/firefox/pingsender
firefox::all:/usr/local/firefox/plugin-container
firefox::all:/usr/local/firefox/libmozwayland.so
firefox::all:/usr/local/firefox/firefox-bin
firefox::all:/usr/local/firefox/libmozsqlite3.so
firefox::all:/usr/local/firefox/updater
firefox::all:/usr/local/firefox/application.ini
firefox::all:/usr/local/firefox/removed-files
firefox::all:/usr/local/firefox/platform.ini
firefox::all:/usr/local/firefox/libplds4.so
firefox::all:/usr/local/firefox/icons/updater.png
firefox::all:/usr/local/firefox/libnssckbi.so
firefox::all:/usr/local/firefox/liblgpllibs.so
firefox::all:/usr/local/firefox/libsmime3.so
firefox::all:/usr/local/share/applications/firefox.desktop
firefox::all:/usr/local/share/icons/hicolor/64x64/apps/firefox.png
firefox::all:/usr/local/share/icons/hicolor/32x32/apps/firefox.png
firefox::all:/usr/local/share/icons/hicolor/128x128/apps/firefox.png
firefox::all:/usr/local/share/icons/hicolor/48x48/apps/firefox.png
firefox::all:/usr/local/share/icons/hicolor/256x256/apps/firefox.png
firefox::all:/usr/local/share/icons/hicolor/16x16/apps/firefox.png
Exiting due to above errors.
```

(I get this issue even with a `crew remove firefox ; crew install firefox` )

**Comment by supechicken** [comment]
This is due to the revert of #7018, all packages installed after #7018 and before #7090 will break because of inconsistent package names (merging #7096 might fix it)

**Comment by supechicken** [comment]
> This is due to the revert of #7018, all packages installed after #7018 and before #7090 will break because of inconsistent package names (merging #7096 might fix it)

(and reinstalling `firefox` will not work either as `firefox::all.filelist` still exist)

**Comment by saltedcoffii** [comment]
closing as duplicate of #6863
